### PR TITLE
fix(oonimkall.HTTPRequest): `URL` => `Url` to fix iOS

### PR DESCRIPTION
**Pull request by aanorbel**
Closes: https://github.com/ooni/probe/issues/2701

### CODE REVIEW

File: pkg/oonimkall/httpx.go [hunk 1]
```diff
@@ -29,8 +29,8 @@ type HTTPRequest struct {
 	// Method is the MANDATORY request method.
 	Method string
 
-	// URL is the MANDATORY request URL.
-	URL string
+	// Url is the MANDATORY request URL.
```
Comment (aanorbel) [inline pkg/oonimkall/httpx.go:32]:
```suggestion
	// Url is the MANDATORY request URL. See https://github.com/ooni/probe/issues/2701
```

File: pkg/oonimkall/httpx.go [hunk 2]
```diff
@@ -29,8 +29,8 @@ type HTTPRequest struct {
 	// Method is the MANDATORY request method.
 	Method string
 
-	// URL is the MANDATORY request URL.
-	URL string
+	// Url is the MANDATORY request URL. See https://github.com/ooni/probe/issues/2701
```
Comment (bassosimone) [inline pkg/oonimkall/httpx.go:32]:
```suggestion
	// Url is the MANDATORY request URL.
	//
	// This variable MUST be named "Url" not "URL"; see https://github.com/ooni/probe/issues/2701.
```

File: pkg/oonimkall/httpx.go [hunk 3]
```diff
@@ -29,8 +29,10 @@ type HTTPRequest struct {
 	// Method is the MANDATORY request method.
 	Method string
 
-	// URL is the MANDATORY request URL.
-	URL string
+	// Url is the MANDATORY request URL.
+	//
+	// This variable MUST be named "Url" not "URL"; see https://github.com/ooni/probe/issues/2701.
```
Comment (bassosimone) [inline pkg/oonimkall/httpx.go:34]:
```suggestion
	// Note: this field MUST be named "Url" not "URL"; see https://github.com/ooni/probe/issues/2701.
```

**Review by bassosimone — APPROVED** [review]
🐳

**Pull request merged**